backend/README.md

```diff
@@ -151,3 +151,50 @@ $(cd alembic && alembic upgrade head)
 serverless wsgi serve
 ```
 
+### Deployment 
+
+This API is deployed as a serverless application on AWS. The entire stack deployment is handled but the serverless configuration. **However**, the database migrations are not done as part of the deployment and have to be run separately from the serverless deployment (see details below).
```

Comment:
The entire stack is handled `with` (?) the serverless configuration

Reply:
Haha, nice. Thanks for the proofread!

Yeah, I deployed the current dev environment using these configs. I still need to setup a CloudFront distribution to front the S3 bucket but will do that as a one-time out-of-band setup.